functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/jobs/JobsSpec.groovy

```diff
@@ -413,6 +413,34 @@ class JobsSpec extends SeleniumBase {
             nextUi << [false, true]
     }
 
+    def "job workflow alphaUi"() {
+        when:
+            def config = [
+                    [
+                            "key": "rundeck.feature.alphaUi.enabled",
+                            "value": true
+                    ],
+            ]
+            enableFeatureFlag(config)
+            waitFeatureFlag("rundeck.feature.alphaUi.enabled")
+            def jobCreatePage = page JobCreatePage, SELENIUM_BASIC_PROJECT
+            jobCreatePage.nextUi=true
+            jobCreatePage.go()
+        then:
+            jobCreatePage.tab JobTab.WORKFLOW click()
+            jobCreatePage.waitForTextToBePresentBySelector(By.xpath("//section[@id='workflowContent2']//div[contains(@class, 'control-label')]"), "Workflow",60)
+        expect:
+            jobCreatePage.workflowAlphaUiContainer.isDisplayed()
+        cleanup:
+            def cleanUpConfig = [
+                    [
+                            "key": "rundeck.feature.alphaUi.enabled",
+                            "value": false
+                    ],
+            ]
+            enableFeatureFlag(cleanUpConfig)
+    }
```

Comment:
I'm unsure if this file is the best place for this test. The same question also applies to the enable/wait for flag methods in the baseContainer. 

I wanted to add a toggle method in the JobCreatePage to do this but then it wouldn't see the methods as it doesn't extend it (it extends basePage), but moving them there also seemed wrong as it would need to bring in the client logic.